install.sh

```diff
@@ -444,12 +444,14 @@ distro() {
   if [ -f /etc/os-release ]; then
     (
       . /etc/os-release
-      case "$ID" in opensuse-*)
-        # opensuse's ID's look like opensuse-leap and opensuse-tumbleweed.
-        echo "opensuse"
-        return
-        ;;
-      esac
+      if [ -n "${ID_LIKE-}" ]; then
+        for distro_like in $ID_LIKE; do
```

Comment:
Let's use lowercase `id_like` here.

Reply:
`$ID_LIKE` is case-senstiive as it comes from `/etc/os-release` - perhaps we should uppercase `distro_like` instead?

Reply:
it's idiomatic shell to use lowercase variables for function local variables. I should have clarified, `distro_like` should be renamed to `id_like`.

Reply:
Ah, thanks for the clarification! And done! :+1: